devdoc/umockcallrecorder_requirements.md

```diff
@@ -200,3 +201,24 @@ extern int umockcallrecorder_fail_call(UMOCKCALLRECORDER_HANDLE umock_call_recor
 **SRS_UMOCKCALLRECORDER_01_050: [** If index is invalid, umockcallrecorder_fail_call shall return a non-zero value. **]**
 
 **SRS_UMOCKCALLRECORDER_01_051: [** If umockcall_set_fail_call fails, umockcallrecorder_fail_call shall return a non-zero value. **]**
+
+
+## umockcallrecorder_can_call_fail
+
+```c
+extern int umockcallrecorder_can_call_fail(UMOCKCALLRECORDER_HANDLE umock_call_recorder, size_t index, int* can_call_fail);
+```
+
+
+**SRS_UMOCKCALLRECORDER_31_056: [** If umock_call_recorder or call_can_fail is NULL, umockcallrecorder_can_call_fail shall return a non-zero value. **]**
+
+**SRS_UMOCKCALLRECORDER_31_057: [** If index is invalid, umockcallrecorder_can_call_fail shall return a non-zero value. **]**
```

Comment:
>invalid [](start = 48, length = 7)

Consider stating greater or equal to the current expected calls count (keyword being the fact that we evaluate it against the expected call count.